demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1923,214 +1890,211 @@ static OtaHttpStatus_t httpRequest( uint32_t rangeStart,
+
+    xCommandStatus = MQTTAgent_Publish( &xGlobalMqttAgentContext, &publishInfo, &xCommandParams );
+    configASSERT( xCommandStatus == MQTTSuccess );
 
     xTaskNotifyWait( 0,
                      0,
                      NULL,
                      pdMS_TO_TICKS( MQTT_AGENT_MS_TO_WAIT_FOR_NOTIFICATION ) );
 
-    if( commandContext.xReturnStatus != MQTTSuccess )
+    if( xCommandContext.xReturnStatus != MQTTSuccess )
     {
-        LogError( ( "Failed to send PUBLISH packet to broker with error = %u.", commandContext.xReturnStatus ) );
-        otaRet = OtaMqttPublishFailed;
+        LogError( ( "Failed to send PUBLISH packet to broker with error = %u.", xCommandContext.xReturnStatus ) );
+        xMqttStatus = OtaMqttPublishFailed;
     }
     else
     {
         LogInfo( ( "Sent PUBLISH packet to broker %.*s to broker.\n\n",
-                   topicLen,
-                   pTopic ) );
+                   usTopicLen,
+                   pcTopic ) );
     }
 
-    return otaRet;
+    return xMqttStatus;
 }
+/*-----------------------------------------------------------*/
 
-static OtaMqttStatus_t prvMqttUnSubscribe( const char * pTopicFilter,
-                                           uint16_t topicFilterLength,
-                                           uint8_t qos )
+static OtaMqttStatus_t prvMqttUnSubscribe( const char * pcTopicFilter,
+                                           uint16_t usTopicFilterLength,
+                                           uint8_t ucQOS )
 {
-    OtaMqttStatus_t otaRet = OtaMqttSuccess;
-    MQTTStatus_t commandStatus;
-    CommandInfo_t commandParams = { 0 };
-    CommandContext_t commandContext = { 0 };
-    MQTTSubscribeInfo_t subscription = { 0 };
-    MQTTAgentSubscribeArgs_t subscribeArgs = { 0 };
-
-    subscription.qos = qos;
-    subscription.pTopicFilter = pTopicFilter;
-    subscription.topicFilterLength = topicFilterLength;
-    subscribeArgs.numSubscriptions = 1;
-    subscribeArgs.pSubscribeInfo = &subscription;
-
-
-    commandParams.blockTimeMs = MQTT_AGENT_SEND_BLOCK_TIME_MS;
-    commandParams.cmdCompleteCallback = prvMQTTSubscribeCompleteCallback;
-    commandParams.pCmdCompleteCallbackContext = &commandContext;
-    commandContext.xTaskToNotify = xTaskGetCurrentTaskHandle();
-    commandContext.pArgs = &subscribeArgs;
-    commandContext.xReturnStatus = MQTTSendFailed;
+    OtaMqttStatus_t xMqttStatus = OtaMqttSuccess;
+    MQTTStatus_t xCommandStatus;
+    CommandInfo_t xCommandParams = { 0 };
+    CommandContext_t xCommandContext = { 0 };
+    MQTTSubscribeInfo_t xSubscription = { 0 };
+    MQTTAgentSubscribeArgs_t xSubscribeArgs = { 0 };
+
+    xSubscription.qos = ucQOS;
+    xSubscription.pTopicFilter = pcTopicFilter;
+    xSubscription.topicFilterLength = usTopicFilterLength;
+    xSubscribeArgs.numSubscriptions = 1;
+    xSubscribeArgs.pSubscribeInfo = &xSubscription;
+
+
+    xCommandParams.blockTimeMs = MQTT_AGENT_SEND_BLOCK_TIME_MS;
```

Comment:
This macro is missing a pre-fix

Reply:
Some configs related to MQTT, Agent, HTTP and Network are not pre-fixed for now.